test/networks/views/input_ordering_view.cpp

```diff
@@ -0,0 +1,33 @@
+//
+// Created by benjamin on 11.04.23.
+//
+
+#include <catch2/catch_test_macros.hpp>
+
+#include <fiction/networks/views/input_ordering_view.hpp>
+
+#include <fiction/algorithms/network_transformation/fanout_substitution.hpp>
+#include <fiction/networks/technology_network.hpp>
+#include <utils/blueprints/network_blueprints.hpp>
+
+#include <mockturtle/networks/aig.hpp>
+#include <mockturtle/views/names_view.hpp>
+
+#include <vector>
+
+using namespace fiction;
+
+TEST_CASE("Input_ordering consider all PIs", "[input-ordering]")
```

Comment:
Can we have a couple more test cases here? They don't seem to properly address all functions defined in `input_ordering_view`.